doc/rados/operations/monitoring.rst

```diff
@@ -352,11 +352,11 @@ The output of ``ceph df`` looks like this::
    TOTAL  202 GiB  200 GiB  2.0 GiB   2.0 GiB       1.00
 
    --- POOLS ---
-   POOL                   ID  PGS   STORED   (DATA)   (OMAP)   OBJECTS     USED  (DATA)   (OMAP)   %USED  MAX AVAIL  QUOTA OBJECTS  QUOTA BYTES  DIRTY  USED COMPR  UNDER COMPR
```

Comment:
Instead of removing this, can we make a note that the "DIRTY" field is only relevant, and will be displayed in `ceph df detail` when cache tiering is in use?